addons/mail/static/src/chatter/web_portal/chatter.js

```diff
@@ -276,9 +275,9 @@ export class Chatter extends Component {
     /**
      * Fetch data for the thread according to the request list.
      * @param {import("models").Thread} thread
-     * @param {['activities'|'followers'|'attachments'|'messages'|'suggestedRecipients']} requestList
+     * @param {[string]} requestList
      */
-    load(thread, requestList = ["followers", "attachments", "messages", "suggestedRecipients"]) {
+    load(thread, requestList = this.defaultRequestList) {
```

Comment:
Instead of the default being here, I think it would prefer if we made the param required and we always called it with the list we need.